src/store/modules/app.js

```diff
@@ -73,6 +89,33 @@ const getters = {
 };
 
 const actions = {
+  checkValidKeymap(_, keymap) {
```

Comment:
TODO figure out what the _ is for

Reply:
the object passed with `{state, commit ... }` . So as a convention, because we are not using it, i've put `_`